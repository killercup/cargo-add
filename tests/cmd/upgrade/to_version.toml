bin.name = "cargo-upgrade"
args = ["upgrade", "docopt@1000000.0.0"]
status = "success"
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
