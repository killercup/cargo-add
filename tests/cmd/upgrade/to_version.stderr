    Upgrading docopt v0.8.0 -> v1000000.0.0
